buildspec.yaml

```diff
@@ -15,6 +15,11 @@ phases:
       - DEPLOY_HASH="$(git rev-parse HEAD)"
       # https://makecode.com/cli/staticpkg
       - node node_modules/pxt-core/built/pxt.js staticpkg -r "$DEPLOY_HASH"
+      # We store the built hexfiles in our repo, then copy them into the cache
+      # This prevents us needing the compilation api to build
+      - cp built_hexfiles/* built/packaged/$DEPLOY_HASH/hexcache/
```

Comment:
I ended up fixing the source last time:

8d22dc03f02cdbcbd8bdb067f6a3e5d510bb8bbd

which we could do this time as well (for connect.png). But it seems like there's some fundamental mess going on here, so :man_shrugging:

Reply:
Oh yeah, also, can you do the symlink version instead of the cp? (see what that commit killed)

Reply:
Why not use the npm scripts?